### PR TITLE
chore(react-scheduler): add the Toolbar plugin API reference

### PR DESCRIPTION
**Pull request by MaximKudriavtsev**

### CODE REVIEW

File: packages/dx-react-scheduler/docs/reference/toolbar.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## Import
+
+Use the following statement to import a plugin with embedded theme components:
+
+```js
+import { Toolbar } from '@devexpress/dx-react-scheduler-material-ui';
+```
+
+If you want to use custom components, you can import the themeless plugin:
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/reference/toolbar.md]: You can import the themeless plugin to use custom components.

File: packages/dx-react-scheduler/docs/reference/toolbar.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+------|------|------------
+children? | ReactNode | A React node to be placed in the toolbar.
+
+### Toolbar.FlexibleSpaceProps
+
+Describes properties passed to a component that renders the the toolbar's empty area.
+
+Field | Type | Description
+------|------|------------
+children? | ReactNode | A React node to be placed inside the empty area.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/reference/toolbar.md]: A React node that should be placed inside the empty area?

File: packages/dx-react-scheduler/docs/reference/toolbar.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## Interfaces
+
+### Toolbar.RootProps
+
+Describes properties passed to a component that renders the toolbar's root element.
+
+Field | Type | Description
+------|------|------------
+children? | ReactNode | A React node to be placed in the toolbar.
```
Comment (dirk-pieterse) [inline packages/dx-react-scheduler/docs/reference/toolbar.md]: A React node that should be placed in the toolbar?

File: packages/dx-react-scheduler/docs/reference/toolbar.md (hunk 2 above)
Comment (MaximKudriavtsev) [reply packages/dx-react-scheduler/docs/reference/toolbar.md]: Yes

File: packages/dx-react-scheduler/docs/reference/toolbar.md (hunk 3 above)
Comment (MaximKudriavtsev) [reply packages/dx-react-scheduler/docs/reference/toolbar.md]: Yes

**Pull request merged**